tests/lib/rules/no-extra-parens.js

```diff
@@ -152,7 +154,6 @@ ruleTester.run("no-extra-parens", rule, {
         "var o = { foo: (function() { return bar(); })() };",
         "o.foo = (function(){ return bar(); })();",
         "(function(){ return bar(); })(), (function(){ return bar(); })()",
-        "((function(){ return bar(); })())",
```

Comment:
Why is that? If we fix a bug, I'm sure they would be happy to start getting notified about things that eslint was missing before.